api/v1alpha1/ratelimit_types.go

```diff
@@ -28,8 +28,11 @@ type RateLimitSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
-	// Foo is an example field of RateLimit. Edit RateLimit_types.go to remove/update
-	Foo string `json:"foo,omitempty"`
+	Conditions []string `json:"conditions,omitempty"`
+	MaxValue   int      `json:"max_value,omitempty"`
```

Comment:
Optional value? if optional, it needs the comment `// +optional ` and make the type pointer to know if it has been set or not.

Reply:
Sorry we discussed a similar problem in another PR and after fixing it, I forgot to fix it in this one. `Conditions` and `Variables` should be optional and default to empty. The rest are mandatory. I'll fix this.

Reply:
I thought those 2 were optional in Limitador, but it turns out they're not. I'll leave this as it is now, but I'll probably change this in the Limitador API.

Reply:
Then, you should remove the `omitempty`. Now, if MaxValue is set to it’s default value (which is 0 for the int type), the key itself will be omitted from the JSON object.

Reply:
you can add [kubebuilder validations](https://book.kubebuilder.io/reference/markers/crd-validation.html): 
```
// +kubebuilder:validation:Minimum=1
```

Reply:
Right. I deleted the `omitempty` tags :+1: